src/Compilers/CSharp/Portable/Binder/Binder_Patterns.cs

```diff
@@ -111,8 +117,23 @@ private BoundExpression BindIsPatternExpression(IsPatternExpressionSyntax node,
                 }
             }
 
+            // decisionDag, whenTrueLabel, and whenFalseLabel represent the decision DAG and
+            // branch labels for the original pattern rather than the inner pattern.
```

Comment:
>branch labels for the original pattern rather than the inner pattern. [](start = 15, length = 69)

Is this accurate? If this is for the original pattern then why any flipping is necessary?